src/react/__mocks__/ProgressBarPlugin.js

```diff
@@ -0,0 +1,18 @@
+const Plugin = require('../../core/Plugin')
+
+module.exports = class ProgressBar extends Plugin {
+  constructor (uppy, opts) {
+    super(uppy, opts)
+
+    this.id = opts.id
+    this.type = 'progressindicator'
+  }
+
+  install () {
+    this.opts.onInstall()
+  }
+
+  uninstall () {
+    this.opts.onUninstall()
+  }
+}
```

Comment:
I’m a bit confused. We have an actual `ProgressBar` Uppy React wrapper component that we are testing. Why do we need a mock for it?

Reply:
The actual component is being tested, but the plugin is mocked so that we don't need a full DOM environment to exist. The mock also adds the `onInstall()` and `onUninstall()` things that the test can use to check if the install/uninstall plugin methods were called at the right times.